The Hoarder project uses a very specific version of pnpm (9.0.0-alpha.8) and
will fail to build with other pnpm versions. Instead of adding this pnpm
version to nixpkgs, we override this requirement and use the latest v9 release.

---
--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "turbo": "^2.1.2"
   },
   "prettier": "@hoarder/prettier-config",
-  "packageManager": "pnpm@9.0.0-alpha.8+sha256.a433a59569b00389a951352956faf25d1fdf43b568213fbde591c36274d4bc30",
+  "packageManager": "pnpm",
   "pnpm": {
     "patchedDependencies": {
       "xcode@3.0.1": "patches/xcode@3.0.1.patch"
--- a/pnpm-lock.yaml
+++ b/pnpm-lock.yaml
@@ -1,4 +1,4 @@
-lockfileVersion: '7.0'
+lockfileVersion: '9.0'
 
 settings:
   autoInstallPeers: true
